cvat/apps/engine/models.py

```diff
@@ -430,6 +432,26 @@ class Meta:
     def touch(self) -> None:
         self.save(update_fields=["updated_date"])
 
+class ABCModelMeta(ABCMeta, ModelBase):
+    pass
+
+class FileSystemRelatedModel(models.Model, metaclass=ABCModelMeta):
```

Comment:
Does this actually need to inherit from `Model`? It's got no fields.

Reply:
> Does this actually need to inherit from Model? It's got no fields.

There were several new fields, but during fixing comments, those fields were deleted and I forgot to delete the base class